core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/HttpWorkspaceManagerDAO.scala

```diff
@@ -101,4 +105,10 @@ class HttpWorkspaceManagerDAO(baseWorkspaceManagerUrl: String)(implicit val syst
   override def enumerateDataRepoSnapshotReferences(workspaceId: UUID, offset: Int, limit: Int, accessToken: OAuth2BearerToken): ResourceList = {
     getResourceApi(accessToken).enumerateResources(workspaceId, offset, limit, ResourceType.DATA_REPO_SNAPSHOT, StewardshipType.REFERENCED)
   }
+
+  def enableApplication(workspaceId: UUID, applicationId: String, accessToken: OAuth2BearerToken): WorkspaceApplicationDescription = {
```

Comment:
Do we need to worry about cleaning up anything on workspace deletion?

And are the code coverage warnings legitimate?

Reply:
Re: cleaning up, short answer: yes, but I want to push it out of scope for this PR. I've dropped a follow-up ticket [here](https://broadworkbench.atlassian.net/browse/WOR-228) that we can refine when ready.

Re: Codecov warnings, yes. This is an artifact of there not being any real coverage on the concrete HttpWorkspaceManagerDAO class (similar to the situation we are in with the HttpGoogleServicesDAO class, not good). I'll look into adding a test as part of this PR, thanks for calling it out.